domain/src/main/kotlin/com/henriquebarucco/ssbc/sensor/SensorGateway.kt

```diff
@@ -1,7 +1,12 @@
 package com.henriquebarucco.ssbc.sensor
 
+import com.henriquebarucco.ssbc.sensor.dto.FetchSensorDto
+import com.henriquebarucco.ssbc.sensor.dto.SensorPageDto
+
 interface SensorGateway {
     fun save(sensor: Sensor): Sensor
 
     fun ofId(sensorId: SensorId): Sensor?
+
+    fun getSensor(fetchSensorDto: FetchSensorDto): SensorPageDto

```

Comment:
acho redundante o nome Sensor no get, visto que o gateway é explicito de sensor.

que tal alterar o método para fetch?


```suggestion
    fun fetch(fetchSensorDto: FetchSensorDto): SensorPageDto
```